timon/db/backends.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.db.backends
+"""
+Summary: Backend Classes to use in dbstore
+"""
+# #############################################################################
+import logging
+from datetime import datetime
+from queue import Queue
+from threading import Event
+from threading import Lock
+
+from peewee import SqliteDatabase
+
+logger = logging.getLogger(__name__)
+
+
+class BaseBackend():
+    """
+    This class is unuseful, but helps to know which funcs must be implemented
```

Comment:
perhaps:
This classe is just an abstract base class and methods need to be implemented.

but if not at least unuseful -> useless